src/components/CollectionSummary/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { ArrowDropDown, ArrowDropUp } from '@mui/icons-material';
+import { Box, Button, Fade, Typography } from '@mui/material';
+import { useMemo, useState } from 'react';
+
+export interface CollectionSummaryProps {
+	items: string[];
+	itemsType: string;
+	expanded?: boolean;
+	maxVisibleItemCount?: number;
+}
+
+export const CollectionSummary = ({
+	expanded,
+	items,
+	itemsType,
+	maxVisibleItemCount,
+}: CollectionSummaryProps) => {
+	const [isExpanded, setIsExpanded] = useState(expanded ?? false);
+	const stringifyItems = useMemo(() => {
+		if (!items.length) {
+			return null;
+		}
+		if (maxVisibleItemCount && items.length > maxVisibleItemCount) {
+			return `${items.slice(0, maxVisibleItemCount).join(', ')}, ...`;
+		}
+		return items.join(', ');
+	}, [items, maxVisibleItemCount]);
+
+	if (!items.length) {
+		return null;
+	}
+
+	return (
+		<Box>
+			{items.length === 1 ? (
+				<Typography variant="body1" color="text.secondary">
```

Comment:
Same as the other comment

Reply:
for this one I would use `body`, which is the default variant so no need to specify it :)